packages/x-components/src/components/panels/__tests__/base-header-toggle-panel.spec.ts

```diff
@@ -87,7 +87,7 @@ describe('testing BaseHeaderTogglePanel component', () => {
     expect(wrapper.find(getDataTestSelector('toggle-status')).text()).toBe('open');
 
     await wrapper.find(getDataTestSelector('toggle-button')).trigger('click');
-
+    wrapper.vm.$nextTick();
```

Comment:
For me, the tests pass without this line. If this is here for something that I can't replicate, then remove the `await` in the line 89 and put it here instead.